tests/src/test/scala/cats/tests/ListWrapper.scala

```diff
@@ -43,25 +42,27 @@ object ListWrapper {
 
   def eqv[A : Eq]: Eq[ListWrapper[A]] = Eq.by(_.list)
 
-  val traverse: Traverse[ListWrapper] = {
+  val traverse: Traverse[ListWrapper] with InvariantSemigroupal[ListWrapper] = {
```

Comment:
I'm not sure that this return type should be changed. There may be cases where we are writing tests to ensure that an implicit instance is found when we have only a `Traverse` instance for `F`, and those tests would no longer be valid.

Reply:
We could always add a separate member if you want want that has this full type returned.

Reply:
You're right, I'll change it!